src/EventListener/ExceptionListener.php

```diff
@@ -79,6 +79,10 @@ public function onConsoleError(ConsoleErrorEvent $event): void {
       }
     }
 
+    if ($error instanceof \TypeError && str_contains($error->getMessage(), 'AcquiaCloudApi\Response')) {
+      $newErrorMessage = 'Cloud API returned an unexpected data type; this could indicate a problem with your Cloud application.';
```

Comment:
What about: "_Cloud Platform API returned an unexpected data type. This is not an issue with Acquia CLI but could indicate a problem with your Cloud Platform application._"?